readme.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🎯 Catalog of my things](#-catalog-of-my-things)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Video presentation](#video-presentation)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [👋 Show your support ](#-show-your-support-)
+  - [🔭Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🎯 Catalog of my things<a name="about-project"></a>
+
+> This is a console application that is built with ruby. It is used to manage items of an individual in a catalog. It has functionalities that include adding a book, music and game, with different properties attached to these items.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  
+</details>
+
+<!-- prsentation vidéo -->
+
+### Video presentation
+
+[Video](https://drive.google.com/file/d/1GOK9NucJGQKEHaH4i_tv26YqYl6Upfmf/view?usp=sharing)
+

```

Comment:
- [ ] Great work on recording the video but it would be great if you check the requirements of the video because some things were missing in your video :smile: 

1. I have seen that you have told your objectives and responsibilities but it would be great if you show the features each person has done by giving a demo of each feature :+1:
2. It would be great if you show the piece of code which you are very proud of :+1: i saw that you have showed the feature which you are proud of but the requirement was to show the piece of code which you are proud of making :+1: